backend/app/api/routes/openai_conversation.py

```
@@ -0,0 +1,170 @@
+        return APIResponse.success_response(
+            data=OpenAIConversationPublic.model_validate(conversation)
+        )
+    except Exception as e:
+        raise HTTPException(
+            status_code=400, detail=f"Failed to create conversation: {str(e)}"
+        )
+
+
+@router.get(
+    "/list",
+    response_model=APIResponse[List[OpenAIConversationPublic]],
```

Ruff (0.12.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)